client/src/pages/SignUpModal.tsx

```diff
@@ -0,0 +1,82 @@
+import './Modal.scss';
+import React, { useState } from 'react';
+import CommonModal from '../components/organisms/Modal';
+import { useApolloClient } from "@apollo/client";
+import { useDispatch } from "react-redux";
+
+
+function SignUpModal() {
+    const client = useApolloClient();
+    const [email, setEmail] = useState("");
+    const [password, setPassword] = useState("");
+    const [firstName, setFirstName] = useState("");
```

Comment:
For now, the designers don't want to store firstname/lastname of users. could we remove name vars from this component?